package.json

```diff
@@ -21,11 +21,12 @@
     "dist/*",
     "entry-asar/*",
     "!entry-asar/**/*.ts",
```

Comment:
nit: we're publishing some unnecessary `.mts` files from `entry-asar/esm` to npm, this should filter them out:

```suggestion
    "!entry-asar/**/*.{ts,mts}",
```